### PR TITLE
Fix incorrect parsing of region

### PR DESCRIPTION
**Pull request by geraud**

### CODE REVIEW

File: core/src/Network/AWS/Types.hs [hunk 1]
```diff
@@ -439,8 +439,8 @@ instance FromText Region where
         "ap-southeast-2"     -> pure Sydney
         "cn-north-1"         -> pure Beijing
         "us-east-1"          -> pure NorthVirginia
-        "us-west-2"          -> pure NorthCalifornia
-        "us-west-1"          -> pure Oregon
+        "us-west-2"          -> pure Oregon
+        "us-west-1"          -> pure NorthCalifornia
```
Comment (brendanhay) [inline core/src/Network/AWS/Types.hs]: Nice catch :) Thanks!

**Comment by brendanhay** [comment]
:+1:

**Comment by brendanhay** [comment]
Your fix has been released as part of version [0.3.4](hackage.haskell.org/package/amazonka-0.3.4) on Hackage.

**Comment by geraud** [comment]
Thanks a lot for the quick turn around!

**Pull request merged**